bindings/python/README.md

```diff
@@ -24,6 +25,34 @@ Python binding to the [iota-sdk library](/README.md).
 
 ## Getting Started
 
+### Install IOTA SDK via pip
+
+1. (optional) Create a virtual environment and use it. On Linux and macOS, you can run the following commands:
+
+   ```bash
+   python3 -m venv env
+   source env/bin/activate
+   ```
+
+   If you are using Windows, you should run the following instead:
+
+   ```bash
+   python3 -m venv env
+   .\env\Scripts\activate`
```

Comment:
I know it was below already like this, but can you remove the ` at the end please? (also below)

Reply:
I removed it from there.